.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Integration Tests
+on:
+  workflow_dispatch
+
+jobs:
+  # Runs integration tests for each os
+  run-integration-tests:
+    name: Run integration test
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ windows-latest, ubuntu-latest, macos-latest, macos-13 ]
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Run integration tests
+        shell: bash
+        run: |
+          mkdir -p ${HOME}/.ide
+          touch ${HOME}/.ide/.license.agreement
```

Comment:
cant we do such things in `all-tests.sh`?
I prefer to do as little logic into workflow files as possible because they are the hardest part to test & maintain.

Reply:
Moved it to `all-tests.sh` now.